srcpkgs/opencv/template

```diff
@@ -14,7 +14,7 @@ homepage="http://opencv.org"
 license="BSD-3-Clause"
```

Comment:
license= above homepage=

homepage= is now on https

Reply:
changed it. Added ck skip to not waste another 5 hours of travis-ci.

Reply:
forgot to push ?

Reply:
forgot to add